netCDF4/_netCDF4.pyx

```diff
@@ -1421,6 +1508,10 @@ compound types defined for the `netCDF4.Group` or `netCDF4.Dataset` to instances
 variable-length types defined for the `netCDF4.Group` or `netCDF4.Dataset` to instances 
 of the `netCDF4.VLType` class.
 
+**`enumtypes`**: The `enumtypes` dictionary maps the names of
```

Comment:
The 'cmptypes', 'vltypes' and 'enumtypes' attributes are dicts similar to 'dimensions' and 'variables'.  They allow the user to quickly find out what's in the file.  If we didn't provide this, you would have to look at the data type of each variable to tell what kind of exotic data types were being used.